Src/Application/Score/MaritalStatus/Queries/GetMaritalStatusScoreQueryHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Application.Common.Interfaces;
+using Domain.Entities;
+using Mapster;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Score.MaritalStatus.Queries;
+
+public class GetMaritalStatusScoreQueryHandler : IRequestHandler<GetMaritalStatusScoreQuery, IReadOnlyList<GetMaritalStatusScoreDto>>
+{
+    private readonly IRepository<MaritalStatusScore> _maritalStatusScoreRepository;
+
+    public GetMaritalStatusScoreQueryHandler(IRepository<MaritalStatusScore> maritalStatusScoreRepository)
+    {
+        _maritalStatusScoreRepository = maritalStatusScoreRepository;
+    }
+
+    public async Task<IReadOnlyList<GetMaritalStatusScoreDto>> Handle(GetMaritalStatusScoreQuery request, CancellationToken cancellationToken)
+    {
+        var maritalStatusScores = await _maritalStatusScoreRepository.GetAll()
```

Comment:
I think you should handle as no tracking inside the get All method or add another method inside repository that has AsNoTracking() because in getting data we don't change anything and it is just for reading @FarzanehMG

Reply:
I didn't catch your meaning. @mahdignb